src/ora-dashboard/data-table/DataTable.jsx

```diff
@@ -1,67 +1,118 @@
 import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 import React, { useEffect } from 'react';
+import { Table } from '@edx/paragon';
 import { useDispatch } from 'react-redux';
 import { useParams } from 'react-router';
-import LoadingOrValue from '../../components/value-or-loading/ValueOrLoading';
 import messages from './messages';
 import { fetchOraReports } from '../data/thunks';
 import { oraDataShape } from '../../data/constants';
+import EmbedORAModal from '../embed-ora-modal/EmbedORAModal';
+
+/**
+ * Sort implementation for Paragon Table
+ * @param {any} firstElement
+ * @param {any} secondElement
+ * @param {string} key
+ * @param {string} direction
+ */
+const sort = function sort(firstElement, secondElement, key, direction) {
+  const directionIsAsc = direction === 'asc';
+
+  if (firstElement[key] > secondElement[key]) {
+    return directionIsAsc ? 1 : -1;
+  } if (firstElement[key] < secondElement[key]) {
+    return directionIsAsc ? -1 : 1;
+  }
+  return 0;
+};
 
 function DataTable({ intl, data }) {
   const { courseId } = useParams();
   const dispatch = useDispatch();
   useEffect(() => {
-    Object.keys(data).map((blockId) => data[blockId].status || dispatch(fetchOraReports(courseId, blockId)));
+    if (Object.keys(data).length > 0) {
+      const blockId = Object.keys(data)[0];
+      if (!data[blockId].status) {
+        dispatch(fetchOraReports(courseId, blockId));
+      }
+    }
   }, [courseId, data]);
+
+  // create a copy of data for sortable Table
+  const sortableData = Object.values(data).slice().map(item => ({ ...item, actions: (<EmbedORAModal usageKey={item.id} title={item.name} buttonText="Manage" />) }));
```

Comment:
I think the action here should only show up based on certain criterion, such as the availability of gradable items and staff access.

Reply:
Yeah, but to accommodate this condition we need ``staff_assessment`` flag available. Details on the ticket.

Reply:
@xitij2000, as we will be creating instructor API's to support this on MFE, I guess this ``Manage`` button will not be here anymore. It will be the same as the current instructor dashboard (clicking on numbers in ``Staff`` column). I will do that change after the APIs are created.